<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter WEkEO Credentials</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing"><number>12</number></property><item>
    <widget class="QLabel" name="labelUsername">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditUsername" />
   </item>
   <item>
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    <property name="leftMargin"><number>6</number></property><property name="topMargin"><number>6</number></property><property name="bottomMargin"><number>6</number></property></layout>
   </item>
   <item>
    <widget class="QLabel" name="labelRegister">
     <property name="text">
      <string>
To register for a free account with WEkEO and gain access to its services, go to the 
&lt;a href="https://wekeo.copernicus.eu/register"&gt;registration page&lt;/a&gt;.
      </string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="text">
      <string>Note: Username and password will be saved as plain text in your user folder.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources />
 <connections />
</ui>